src/main/resources/view/PersonListCard.fxml

```diff
@@ -28,9 +28,20 @@
         <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
+      <HBox spacing="20">
```

Comment:
I think it would look nicer if each attribute is on a different line, except the rate, owe and paid

Reply:
You have a good point. I will make the changes. Thank you!